pvlib/pvsystem.py

```diff
@@ -2195,7 +2195,8 @@ def _parse_raw_sam_df(csvdata):
     return df
 
 
-def sapm(effective_irradiance, temp_cell, module):
+def sapm(effective_irradiance, temp_cell, module, reference_temperature=25,
```

Comment:
```suggestion
def sapm(effective_irradiance, temp_cell, module, *, reference_temperature=25,
```
Consider using kwarg-only parameters to enforce readability of code. For future issues debugging, ...
Completely optional suggestion, it's not a common trend in pvlib.

Reply:
Quick lookup on the internet and you will have a ton of information more extensive, but here is a short example, where all statements are expected to print `Hola Dax`. The star means the following arguments are only allowed to be passed in as keyword-only (parameter_name=value):

```python
def hola_dax(greet, name):
    print(f"{greet} {name}")

# allowed uses
hola_dax("Hola", "Dax")
hola_dax("Hola", name="Dax")
hola_dax(greet="Hola", name="Dax")
hola_dax(name="Dax", greet="Hola")

# however
def hola_dax(greet, *, name):
    print(f"{greet} {name}")
# only allows using the forms:
hola_dax("Hola", name="Dax")
hola_dax(greet="Hola", name="Dax")
# this raises an error:
hola_dax("Hola", "Dax")  # TypeError: hola_dax() takes 1 positional argument but 2 were given

# and the form
def hola_dax(*, greet, name):
    print(f"{greet} {name}")
# only allows:
hola_dax(greet="Hola", name="Dax")
hola_dax(name="Dax", greet="Hola")
```
This way, you can always enforce people to make readable code.